graspologic/embed/case.py

```diff
@@ -202,7 +202,7 @@ def _get_tuning_parameter(
 
         # just use the ratio of the leading eigenvalues for the
         # tuning parameter, or the closest value in its possible range.
-        self.alpha_ = np.float_(L_top / YYt_top)
```

Comment:
@loftusa use of `np.float_` was causing issues with mypy - if there is some reason you think this call is necessary, can you propose an alternative fix ASAP?

Reply:
I think if I called it, there may have been a reason, but I'm not sure what it was at this point. I'd be down to just change it to a normal `float` call and see if anything happens

Reply:
and I am guessing both of those start as float anyway? regardless feel free to propose a fix and test that it doesnt mess up mypy if you think the change should be something else

Reply:
Woops, didn't see this was a PR.
Yeah, this change should be fine